src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -473,8 +473,7 @@ class GroupReadsByUmi
     // Output the reads in the new ordering
     logger.info("Assigning reads to UMIs and outputting.")
     val outHeader = header.clone()
-    outHeader.setSortOrder(SortOrder.unsorted)
-    outHeader.setGroupOrder(GroupOrder.query)
+    SamOrder.TemplateCoordinate.applyTo(outHeader)
```

Comment:
Just so I am clear, it was the sub-sort that was missing?

Reply:
Yeah.